readinglist/errors.py

```diff
@@ -85,6 +87,23 @@ def __init__(self, **kwargs):
         super(HTTPInternalServerError, self).__init__(**kwargs)
 
 
+class HTTPServiceDeprecated(PyramidHTTPGone):
+    """A HTTPGone formatted in JSON."""
+
+    def __init__(self, **kwargs):
+        kwargs.setdefault('body', format_error(
+            410,
```

Comment:
In other parts of the project, we used PyramidHTTPGone.code and PyramidHTTPGone.title